extensions/cloud/portability-cloud-local/src/test/java/org/datatransferproject/cloud/local/LocalJobStoreTest.java

```diff
@@ -16,30 +16,31 @@
 
 package org.datatransferproject.cloud.local;
 
-import static org.junit.Assert.assertTrue;
-
 import com.google.common.collect.ImmutableMap;
 import com.google.common.truth.Truth;
-import java.util.Map;
 import org.junit.Test;
 
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.Assert.assertTrue;
```

Comment:
It looks like adding the import has rearranged your imports but this doesn't match the google-java-format. It might be worth running:
`java -jar google-java-format-1.7-all-deps.jar -i extensions/cloud/portability-cloud-local/src/main/java/org/datatransferproject/cloud/local/LocalJobStore.java extensions/cloud/portability-cloud-local/src/test/java/org/datatransferproject/cloud/local/LocalJobStoreTest.java portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/storage/JobStore.java portability-transfer/src/main/java/org/datatransferproject/transfer/PortabilityInMemoryDataCopier.java`

Reply:
Done